kind/cmd/kind/delete/delete.go

```diff
@@ -50,12 +50,12 @@ func run(flags *flags, cmd *cobra.Command, args []string) {
 	// TODO(bentheelder): make this more configurable
 	ctx, err := cluster.NewContext(flags.Name)
 	if err != nil {
-		glog.Error("Failed to create cluster context! %v", err)
+		log.Errorf("Failed to create cluster context! %v", err)
```

Comment:
do you want to switch these to `log.Fatalf` as well?

Reply:
sure, will do. I also want to take a look at the actual logging patterns more broadly in follow up(s), EG I'm debating the merits of `WithError` for cli tools for example.